src/components/app-header/app-header.js

```diff
@@ -3,11 +3,11 @@ import React from 'react';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
-import ShoppingBasketIcon from '@material-ui/icons/ShoppingBasket';
 import { useStyles } from './app-header.styles';
 import Language from '../../containers/language';
 import NavbarLeft from '../../containers/navbar-left';
-import Cabinet from '../cabinet';
+import Cabinet from '../../containers/cabinet';
+import CartIcon from '../../containers/cart-header';
```

Comment:
I have seen these changes in another merge request related to the cart. Please, rollback them here because this merge request is about the wishlist, not the cart. Can you do that?